packages/adminrouter/extra/src/Makefile.common

```diff
@@ -109,21 +111,26 @@ api-docs: clean-devkit-container devkit
 	done
 
 .PHONY: check-api-docs
+DOCS_TEXT := See https://github.com/dcos/dcos/tree/master/packages/adminrouter/extra/src\#endpoints-documentation for more info.
 check-api-docs: api-docs
-	@git update-index --refresh || (echo 'Found local changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
-	@git diff-files --quiet || (echo 'Found unstaged changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
-	@git ls-files --exclude-standard --others || (echo 'Found untracked and unignored changes. Please run `make api-docs` and commit the result.' >&2 && exit 1)
+ifneq ($(shell git update-index --refresh; echo $$?),0)
+	$(error "Found local changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)")
+endif
+ifneq ($(shell git diff-files --quiet; echo $$?),0)
+	$(error 'Found unstaged changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)')
+endif
+ifneq ($(shell git ls-files --exclude-standard --others; echo $$?),0)
+	$(error 'Found untracked and unignored changes. Please run `make api-docs` and commit the result. $(DOCS_TEXT)')
+endif
 	@echo 'No changes found -- Admin Router API docs are up to date.' >&2
 
 .PHONY: lint
 lint: clean-devkit-container devkit
-	#FIXME - split it into two targets
-	docker run \
+	@#FIXME - split it into two targets
```

Comment:
Uhh, this is a very old comment that should have been removed long time ago - IIRC earlier I was also calling here pylint, but it was too problematic and was removed.

Fixing!